exampleSite/content/installation/configuration/_index.en.md

```diff
@@ -24,4 +24,6 @@ Note that some of these parameters are explained in details in other sections of
   # Call to action is default enabled, if you want to disable it. just change the 
   enable = false
   # You can change banner title and other text from the config file.
-```
\ No newline at end of file
+```
+
+{{< embed-pdf pdfName="sample.pdf">}}
```

Comment:
I think we can include this example at bottom of the [Elements](https://unicef.github.io/inventory-hugo-theme/installation/elements/) page to show off what the theme is capable of.

Reply:
Yeah, that makes sense. I will change it.

Reply:
Fixed in commit 46f14ed73cd3e76546a1718415972bc68d552abe.